### ISSUE TITLE
Power to veto own proposed law

### ISSUE THREAD

**Issue by Dave247** [StrangeLoopGames/EcoSuggestions#1682]
An interesting idea I thought of, is to have it possible to set it so the user who proposed the law is able to veto their own proposed law.

The reason for this being often someone will propose a law, realise that after when someone points it out they made a mistake, and wish to amend it but there isn't an easy way to do that without having to go through the whole voting process and waiting for it to fail naturally, either by reaching the threshold of enough people voting or by timeout.

I can't really why of a reason as to why it would be abused either. It would be always assumed that the proposer would support the law they themselves would be putting up, as them vetoing it would be the exact same result as if it was to fail naturally, with the only difference being that it wouldn't take as long and thus be a net benefit to all involved.

While being able to select vetoers' by individual user, demographic, or title, none of those would work here as they would all be too broad as the desire here is to only be able to veto a law that you yourself proposed, which I can't think of any way that can be done with the current system.

**Comment by MystLeissa** [comment]
Good idea. Makes sense to me, To be perfectly honest, even with laws you may have proposed that are correct, you may suddenly come to a realization are no longer neccessary by the time they would be passed; Allowing the "Withdrawl" of a law is a good way to allow it to become more dynamic. As for what it would effect negatively, remember that all laws (proposed or otherwise) are technically made reference to in the Eco Database, without some kind of trash removal the database could get very bloated by griefers.
